src/components/Counter/Counter.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+export const Counter = ({ maxValue, minValue = 0, count, setCount }) => {
+  return (
+    <>
+      <div className="options-item-buttons">
+        <button
+          className="options-button options-minus-button_js"
+          type="button"
+          disabled={count === minValue}
+          onClick={() => setCount(count - 1)}
+        >
+          -
+        </button>
+        <span className="options-counter-number">{count}</span>
+        <button
+          className="options-button options-plus-button_js"
+          type="button"
+          disabled={count === maxValue}
+          onClick={() => setCount(count + 1)}
+        >
+          +
+        </button>
+      </div>
+    </>
+  );
+};
+
+Counter.propTypes = {
+  maxValue: PropTypes.number,
+  minValue: PropTypes.number,
+};
```

Comment:
You added `propTypes`, well done 🎉 
The `Counter` component has 4 props:
- `maxValue`
- `minValue`
- `count`
- `setCount`

But you added only 2 to `propTypes`
```suggestion
Counter.propTypes = {
  maxValue: PropTypes.number,
  minValue: PropTypes.number,
  count: PropTypes.number,
  setCount: PropTypes.func,
};
```